documentation/guide-jwt.asciidoc

```diff
@@ -64,19 +64,28 @@ To allow a client to login with username and password to get a JWT for sub-seque
 
 [source,java]
 ----
+  
   @Bean
-  public JwtLoginFilter getJwtLoginFilter() throws RuntimeException {
+  public JwtLoginFilter getJwtLoginFilter() throws Exception {
```

Comment:
I still do not see why we need this throws declaration at all here, but not so important.

Reply:
here authenticationManager() throws exception

Reply:
OK. Then here it is required. The getJwtAuthenticationFilter() does not throw an exception, however...